src/components/atoms/CardInfoWithCount/CardInfoWithCount.tsx

```diff
@@ -0,0 +1,38 @@
+import * as styles from "./CardInfoWithCount.css";
+
+import { ThumbsUp, ThumbsDown, MessageSquare } from "react-feather";
+
+export const CardInfoWithCount: React.FC<{
+  writer: string;
+  like: number;
+  dislike: number;
+  comment: number;
+}> = ({ writer, like, dislike, comment }) => {
+  return (
+    <div className={styles.contentInfo}>
+      <div className={styles.infoes}>
+        <div className={styles.writer}>{writer}</div>
+      </div>
+      <div className={styles.count}>
+        <div className={styles.likes}>
+          <div className={styles.likeIcon}>
+            <ThumbsUp size={14} />
+          </div>
```

Comment:
icon을 다시 `div`로 묶어줄 필요 없어 보입니다! size는 icon에 바로 넣어주고 있고, color는 `styles.likes`에서 넘어오고 있네요. 아래쪽 코드들도 마찬가지로 수정 부탁드립니다 :)
```suggestion
          <ThumbsUp size={14} />
```